README.md

```diff
@@ -38,6 +43,10 @@ Then visit http://localhost:3000
 
 Managed by [asdf](https://github.com/asdf-vm/asdf)
 
+## Customization
+
+The LikeDAO is a generic DAO platform that does not only work with the LikeCoin chain but also other Cosmos-based chain, the following states the customizable configurations that can be modified to make the app work with one's favourite chain.
```

Comment:
`LikeDAO is a generic DAO platform built for the LikeCoin chain, while other Cosmos-based chains are also supported. Follow the instructions below to modify necessary configurations to adopt LikeDAO to other chains.`

& link to a summarized page for customization?